index.d.ts

```diff
@@ -0,0 +1,9 @@
+type ClassValue = string | number | ClassDictionary | ClassArray;
+
+interface ClassDictionary {
+	[id: string]: boolean;
+}
+
+interface ClassArray extends Array<ClassValue> { }
+
+export default function(...classes: ClassValue[]): string;
```

Comment:
OK Cool good to know.

Looking at your definition on DefinitelyTyped, what is the purpose of the `classnames` module wrapped around the export?

```
declare module "classnames" {
	export = classNames
}
```